mapit_global/tests/test_import_command.py

```diff
@@ -93,6 +94,16 @@ def example_files(type_and_file_data):
             yield tmp_dir
 
 
+def fake_get(url, **kwargs):
+    if url != 'http://www.loc.gov/standards/iso639-2/ISO-639-2_utf-8.txt':
+        raise Exception("The URL {url} hasn't been faked")
+    local_filename = join(dirname(__file__), 'country-data', 'ISO-639-2_utf-8.txt')
+    fake_response = Mock()
+    fake_response.iter_lines.return_value = open(local_filename, 'rb')
+    return fake_response
+
+
+@patch('mapit_global.management.commands.mapit_global_import.requests')
```

Comment:
I've done that in https://github.com/mysociety/global.mapit.mysociety.org/pull/4/commits/a7f67c978cc32634a384f40cb35807ea58657f37 - is that what you expected? (It seems to work fine.)